dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -1072,64 +1106,77 @@ public Response fixConflicts ( @Context final HttpServletRequest request, @PathP
+            if (whereToFix.equals("local")) {
 
-            	HibernateUtil.startTransaction();
+                HibernateUtil.startTransaction();
                 integrityUtil.fixConflicts(endpointId, integrityTypeToFix);
                 HibernateUtil.commitTransaction();
-                jsonResponse.put( "success", true );
-                jsonResponse.put( "message", "Conflicts fixed in Local Endpoint" );
+                jsonResponse.put("success", true);
+                jsonResponse.put("message", "Conflicts fixed in Local Endpoint");
 
                 IntegrityType[] types = IntegrityType.values();
                 boolean isThereAnyConflict = false;
                 // Check if we still have other conflicts
                 for (IntegrityType integrityType : types) {
-					if (!integrityType.equals(integrityTypeToFix)) {
-	                	List<Map<String, Object>> results = integrityUtil.getIntegrityConflicts(endpointId, integrityType);
-	                	if(!results.isEmpty()) {
-	                		isThereAnyConflict = true;
-	                		break;
-	                	}
-                	}
+                    if (!integrityType.equals(integrityTypeToFix)) {
+                        List<Map<String, Object>> results = integrityUtil
+                                .getIntegrityConflicts(endpointId, integrityType);
+                        if (!results.isEmpty()) {
+                            isThereAnyConflict = true;
+                            break;
+                        }
+                    }
                 }
 
-                if(!isThereAnyConflict)
-                	clearStatus( request, endpointId );
+                if (!isThereAnyConflict)
+                    clearStatus(request, endpointId);
 
-            } else  if(whereToFix.equals("remote")) {
+            } else if (whereToFix.equals("remote")) {
                 integrityUtil.generateDataToFixZip(endpointId, integrityTypeToFix);
 
                 final Client client = RestClientBuilder.newClient();
 
-                PublishingEndPoint endpoint = APILocator.getPublisherEndPointAPI().findEndPointById(endpointId);
+                PublishingEndPoint endpoint = APILocator.getPublisherEndPointAPI()
+                        .findEndPointById(endpointId);
                 String outputPath = ConfigUtils.getIntegrityPath() + File.separator + endpointId;
-                File bundle = new File(outputPath + File.separator + INTEGRITY_DATA_TO_FIX_ZIP_FILE_NAME);
+                File bundle = new File(
+                        outputPath + File.separator + INTEGRITY_DATA_TO_FIX_ZIP_FILE_NAME);
 
                 FormDataMultiPart form = new FormDataMultiPart();
                 form.field("AUTH_TOKEN",
                         PushPublisher.retriveKeyString(
-                                PublicEncryptionFactory.decryptString(endpoint.getAuthKey().toString())));
+                                PublicEncryptionFactory
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "PublicEncryptionFactory"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)